notebooks/Analyse_curated_playlist.ipynb

```diff
@@ -3,22 +3,20 @@
   {
```

Comment:
<p>nice graph! Here's a link to <a href="https://bokeh.pydata.org/en/latest/" target="_blank">bokeh</a> instead.</p><p><br></p><p>How about this code?</p><p><code>foo = pd.series([5,2,3])</code></p><p><code>bah = sorted(list(foo))</code></p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/tomsyouruncle/DJ_suggest/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>